src/backends/torch/torchlib.cc

```diff
@@ -244,91 +289,59 @@ namespace dd
     if (unsupported_model_configuration)
       throw MLLibInternalException("Use of libtorch backend needs either: "
                                    "traced net, protofile or native template");
-    // Create the model
+
+    // FIXME(louis): out of if(bert) because we allow not to specify template
+    // at predict. Should we change this?
     this->_inputc._input_format = "bert";
```

Comment:
default input format is `bert` ? This is weird indeed, though I do understand it's been so "historically".

Reply:
@sileht made a PR about this some time ago #893, I don't know why it stayed like this until now